modules/azure/aks/modules.tf

```diff
@@ -213,3 +213,13 @@ module "velero" {
   azure_client_id                 = var.velero.identity.client_id
   azure_resource_id               = var.velero.identity.resource_id
 }
+
+# Velero
+module "tekton-operator" {
```

Comment:
Does this have anything to do with velero?

Reply:
Fixed!